packages/thumbprint-react/components/Avatar/index.jsx

```diff
@@ -225,7 +217,7 @@ UserAvatar.propTypes = {
         PropTypes.number,
     ]),
     /**
-     * Displays a badge of a checkmark next to the `Avatar`.
+     * DEPRECATED: Displays a badge of a checkmark next to the `Avatar`.
```

Comment:
You can now do `@deprecated` to get it to look nice. (Make sure this branch is up-to-date with `master`.)

Reply:
done